src/sourmash/commands.py

```diff
@@ -792,12 +804,14 @@ def gather(args):
 
     # save CSV?
     if found and args.output:
-        fieldnames = ['intersect_bp', 'f_orig_query', 'f_match',
-                      'f_unique_to_query', 'f_unique_weighted',
-                      'average_abund', 'median_abund', 'std_abund', 'name',
-                      'filename', 'md5', 'f_match_orig', 'unique_intersect_bp',
-                      'gather_result_rank', 'remaining_bp',
-                      'query_filename', 'query_name', 'query_md5', 'query_bp']
+        fieldnames = GatherResult._fields
```

Comment:
👍 !!!

Reply:
oh yay! I wasn't sure if you wanted these explicitly for some reason 😂